### PR TITLE
Aggregated Codecov Reports

### PR DESCRIPTION
**Pull request by dhruvigoyal25**
This PR makes the following changes to AdvantEDGE:
1. Updated entrypoint.sh files for core and sandbox pods to execute services based on codecov mode flag and generate timestamp-based codecov output files.
2. Updated meepctl test to generate aggregated codecov reports from the timestamp-based codecov reports.

### CODE REVIEW

File: .meepctl-repocfg.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ repo:
     # platform ingress configuration
     ingress:
       # host name
-      host: my-platform-fqdn
+      host: 10.3.16.69
```
Comment (dilallkx) [inline .meepctl-repocfg.yaml:41]: Undo this change.

**Review by dilallkx — APPROVED** [review]
Updates look great!
Please address the comment on the repocfg file.

File: .meepctl-repocfg.yaml (hunk 1 above)
Comment (dhruvigoyal25) [reply .meepctl-repocfg.yaml]: Adressed

**Comment by dilallkx** [comment]
Please rebase to the tip of develop for merge.
